cms/static/coffee/spec/main.coffee

```diff
@@ -221,8 +221,11 @@ define([
     "coffee/spec/models/settings_grading_spec", "coffee/spec/models/textbook_spec",
     "coffee/spec/models/upload_spec",
 
-    "coffee/spec/views/metadata_edit_spec", "coffee/spec/views/module_edit_spec",
-    "coffee/spec/views/textbook_spec", "coffee/spec/views/upload_spec",
+    "coffee/spec/views/course_info_spec",
```

Comment:
Wow, it looks like these tests haven't been run in a long time! It seems to have got dropped in this refactoring (which, of course, I code reviewed):

https://github.com/edx/edx-platform/pull/9265